src/Illuminate/Database/Schema/SQLiteBuilder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Illuminate\Database\Schema;
+
+class SQLiteBuilder extends Builder
+{
+    /**
+     * Drop all tables from the database.
+     *
+     * @return void
+     */
+    public function dropAllTables()
+    {
+        $dbPath = $this->connection->getConfig('database');
+
+        if (file_exists($dbPath)) {
+            unlink($dbPath);
```

Comment:
I think this will give problems when someone tries to drop all tables in an in-memory SQLite database. An approach similar to the Postgres solution might be better. 

You can get a list of tables with:
```sql
SELECT name FROM sqlite_master WHERE type='table'
```